fastlane_bot/helpers/routehandler.py

```diff
@@ -592,31 +605,29 @@ def _extract_single_flashloan_token(self, trade_instructions: List[TradeInstruct
         :param trade_instructions: A list of trade instruction objects
         """
 
-        if self.ConfigObj.ARB_CONTRACT_VERSION >= 10:
-            tknin_address = None
-            tknin_address = None
-            if trade_instructions[0].tknin_is_native:
-                tknin_address = self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS
-                tknin_address = self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS
-            elif trade_instructions[0].tknin_is_wrapped:
-                tknin_address = self.ConfigObj.WRAPPED_GAS_TOKEN_ADDRESS
-                tknin_address = self.ConfigObj.WRAPPED_GAS_TOKEN_ADDRESS
-            else:
-                tknin_address = trade_instructions[0].tknin_address
-                tknin_address = trade_instructions[0].tknin_address
-            assert  tknin_address is not None and tknin_address is not None, f"routehandler _extract_single_flashloan_token, tknin_address is {tknin_address} and tknin_address is {tknin_address}, must not be None"
-            flash_tokens = {
-                tknin_address :
-                {
-                "tkn": tknin_address,
-                "flash_amt": trade_instructions[0].amtin_wei,
-                "decimals": trade_instructions[0].tknin_decimals}
-            }
+        is_FL_NATIVE_permitted=False
+        if self.ConfigObj.NETWORK in [self.ConfigObj.NETWORK_ETHEREUM]:
+            is_FL_NATIVE_permitted=True
```

Comment:
```
is_FL_NATIVE_permitted = self.ConfigObj.NETWORK == self.ConfigObj.NETWORK_ETHEREUM
```

Reply:
The suggested change would drastically decrease readability IMO